keybuffer.c

```diff
@@ -83,15 +98,12 @@ void st_key_buffer_push(st_key_buffer_t *buf, uint16_t keycode)
 {
     // Store all alpha chars as lowercase
     const bool shifted = keycode & QK_LSFT;
-    const uint8_t lowkey = keycode & 0xFF;
+    const uint8_t lowkey = keycode & ST_DEFAULT_KEY_ACTION;
```

Comment:
0xFF was correct, cause IS_ALPHA_KEYCODE is testing for unshifted KC_A .. KC_Z

Reply:
Good catch!

In this case, ST_DEFAULT_KEY_ACTION is 0xFF. This potential bug was introduced when I decided to give the constant a name and got over-zealous with the replace function!

Reply:
Correction. It was a bug. Because ST_DEFAULT_KEY_ACTION is 0xffff. I just overwrote the wrong constant. I just didn't catch the bug, because I only tested with lower case presses.